### DISCUSSION TITLE
Ticket open for weeks with out a single response

### DISCUSSION THREAD

**Discussion by Granatomatt** [community/community#122794] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have had a support ticket open for almost a full month with no response. Is GitHub support this bad?

**Comment by hafiz-muhammad** [comment]
They get a lot of other tickets so it takes time for them to respond back.

**Comment by ebndev** [comment]
Hi @Granatomatt,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
